DTS-STN/canadian-dental-care-plan
# AB#4010 -- Changing `dateTime` field to `date`

### Description
`/applicant` endpoint returns `date` field rather than `dateTime` with the corresponding format change for the value.

### Related Azure Boards Work Items
[AB#4010](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4010)

### Checklist

### Test Instructions
Run application locally and navigate to `/en/personal-information`. The user's date of birth should (still) display.

### Additional Notes
I tested this on my local machine with an actual call to Interop's INT instance for `/applicant`
